### PR TITLE
Update for webpack 2.x+ support

### PR DESCRIPTION
**Pull request by chuckdumont**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -17,45 +17,53 @@ var DojoWebpackPlugin = require("dojo-webpack-plugin");	// load locally
 
 var path = require("path");
 var webpack = require("webpack");
+var path = require("path");
```
Comment (ddumontatibm) [inline webpack.config.js]: why require twice?

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ To build the sample, run the following commands in the project root folder
 
 ```
 npm install
-node_modules/.bin/webpack --config webpack.config.js --dojoRoot <dojoRoot>
+node_modules/.bin/webpack --config webpack.config.js --env.dojoRoot <dojoRoot>
```
Comment (ddumontatibm) [inline README.md]: can you include a "script" in the package.json to do this?  
Then you can:
````
npm install
npm run webpack
````

Comment (chuckdumont) [inline README.md]: Well, dojoRoot needs to be specified as a command line arg.  Not sure how to pass that as a parameter to the script

File: webpack.config.js (hunk 1 above)
Comment (chuckdumont) [inline webpack.config.js]: looks like a merge error.

**Pull request merged**